src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1672,15 +1664,9 @@ private boolean validateAuthority(final URL authorityUrl) {
 
             // Set CorrelationId for Instance Discovery
             mDiscovery.setCorrelationId(getRequestCorrelationId());
-            try {
-                boolean result = mDiscovery.isValidAuthority(authorityUrl);
-                Logger.v(TAG, "Finish validating authority:" + authorityUrl + " result:" + result);
-                return result;
-            } catch (Exception exc) {
-                Logger.e(TAG, "Instance validation returned error", "",
-                        ADALError.DEVELOPER_AUTHORITY_CAN_NOT_BE_VALIDED, exc);
-
-            }
+            boolean result = mDiscovery.isValidAuthority(authorityUrl);
```

Comment:
Ok, changed Discovery.isValidAuthority to log and return false instead of throwing and catching exception